extension/src/plots/paths/collect.ts

```diff
@@ -157,6 +159,46 @@ const updateExistingPlotPath = ({
   return acc
 }
 
+const separateDvcYamlPath = (path: string): string[] => {
```

Comment:
[F] I've kept these three functions together to try and encapsulate the logic in a single place.